### DISCUSSION TITLE
HiDPI fractional scaling?

### DISCUSSION THREAD

**Discussion by joshsharp** [lapce/floem#704] [Q&A]
Hi, thanks for Floem! It's nice to have something native for Rust that's so far along and supports keyboard navigation.

Apologies if this is already answered in the docs. I'm developing an app on Windows 10 on a 5k monitor scaled to 225%. I'm pretty sure that the reason everything in my app feels a little small to me by default, and that "pixel" sizes on widgets seem a bit off, is that Floem is scaling the window to 2x only. Is that the case and if so, is there a way I can get proper fractional scaling to 2.25x? I don't mind if I have to set that manually, so long as it's possible somehow.

Thanks!

**Comment by jrmoulton** [comment]
Floem sets the window scale from a value in winit. 

You can manually adjust the scale using https://docs.rs/floem/latest/floem/action/fn.set_window_scale.html

**Comment by joshsharp** [comment]
FWIW I'm pretty sure the default value is incorrect, as on 225% scaled Windows, everything is a little too small. I've noticed this with Lapce as well. Not sure whereabouts the issue is. Iced uses winit AFAIK and doesn't have this issue.
